.secrets.baseline

```diff
@@ -110,7 +110,7 @@
     {
       "path": "detect_secrets.filters.regex.should_exclude_secret",
       "pattern": [
-        "(foo|secret|reset|true|toggle|trackForgotClick|passwordNextButton)",
+        "(foo|secret|reset|true|toggle|trackForgotClick|passwordNextButton|hook)",
```

Comment:
Added this pattern to exclude list because [adding this form](https://github.com/artsy/force/pull/9702/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R208), for whatever reason, was tripping up the tool 🤷‍♂️. Seems more appropriate than using the `pragma: allowlist secret`.

```
error Command failed with exit code 1.
yarn run v1.22.17
$ scripts/detect-secrets.sh hook /Users/ozzievasdi/code/force/.circleci/config.yml /Users/ozzievasdi/code/force/package.json /Users/ozzievasdi/code/force/scripts/detect-secrets.sh
Executing detect-secrets-hook...
ERROR: Potential secrets about to be committed to git repo!

Secret Type: Secret Keyword
Location:    .circleci/config.yml:191

Possible mitigations:
  - For information about putting your secrets in a safer place, please ask in
    #security
  - Mark false positives with an inline `pragma: allowlist secret`
    comment
```